config/config.go

```diff
@@ -307,6 +333,18 @@ func NewAgentConfig(agentIni *File, agentYaml *YamlAgentConfig) (*AgentConfig, e
 	return cfg, nil
 }
 
+// Compile each word in the list into a regex pattern to match against the cmdline arguments
+func CompileStringsToRegex(words []string) []*regexp.Regexp {
+	compiledRegexps := make([]*regexp.Regexp, 0, len(words))
+	for _, word := range words {
+		pattern := `((?i)-{1,2}` + word + `[^= ]*[ =])([^ \n]*)`
```

Comment:
This regex is starting to look pretty gnarly. I'm curious about the benchmarks for how this will perform and what kind of computational cost it'll add to the agent. Can you post benchmarks to the RFC?

Reply:
The RFC is updated with the benchmark considering the matching for different numbers of processes running on the host. I can update it to consider just the avg per regex call if it's better

Reply:
The RFC is updated with the new benchmark values